src/middleware/validation.js

```diff
@@ -80,4 +76,23 @@ export default {
     }
     next();
   },
+  signInValidation(req, res, next) {
+    const loginSchema = Joi.object().keys({
+      email: validationRules.email,
+      password: validationRules.loginPassword
+    });
+    const options = {
+      language: {
+        key: '{{key}} '
+      }
+    };
+    const { error } = Joi.validate(req.body, loginSchema, options);
+    if (error) {
+      return res.status(400).json({
+        error: error.details[0].message
+
+      });
```

Comment:
Same duplication case here for the returned error message. It's the same syntax everywhere. If you could create a variable to hold this value and reuse it accordingly.

Reply:
I think you and Carlos are the ones who designed the validation schema and the rules.

Reply:
Yes by the time I designed it, there was no repeated code, but you can still change it to avoid duplication. Besides, It's best practice.